bigquery/system-test/list_datasets_and_projects_test.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var listDatasetsAndProjectsExample = require("../list_datasets_and_projects")
```

Comment:
Can you rework this file to match https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/storage/system-test/buckets.test.js? The sample will need to export `listDatasets` and `listProjects`